ionoscloud/resource_lan.go

```diff
@@ -63,6 +63,17 @@ func resourceLan() *schema.Resource {
 					},
 				},
 			},
+			"ipv6_cidr_block": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				DiffSuppressFunc: func(_, old, new string, _ *schema.ResourceData) bool {
```

Comment:
Is `auto` the default value in case no value is set?

Reply:
the `auto` value means that the lan will be automatically assigned an ipv6 block that is inside the ipv6 block of the datacenter
if no value is provided, the lan will not have ipv6 enabled